### DISCUSSION TITLE
What do the channel numbers with and without parentheses mean?

### DISCUSSION THREAD

**Discussion by AmiDavidW** [VREMSoftwareDevelopment/WiFiAnalyzer#371] [Q&A] [answered]
On the channel display, we can see, for example, CH 132(138) in the screenshot.
What do 132 and 138 mean?

![image](https://user-images.githubusercontent.com/47610659/156069955-45a50487-0932-46a0-a326-fae8c25f0c0a.png)

**Comment by VREMSoftwareDevelopment** [comment, marked as answer]
Please take a look at 
- [FAQ](https://github.com/VREMSoftwareDevelopment/WiFiAnalyzer#faq)
- [How channel graph displays 20/40/80/160 MHz channel width?](https://github.com/VREMSoftwareDevelopment/WiFiAnalyzer/issues/64)
